ikvmc/AssemblyResolver.cs

```diff
@@ -81,7 +86,7 @@ internal void Init(Universe universe, bool nostdlib, IList<string> references, I
 			}
 			else
 			{
-				mscorlibVersion = universe.Load("mscorlib").GetName().Version;
+				mscorlibVersion = universe.Load(Universe.CoreLibName).GetName().Version;
```

Comment:
I thought (may be misremebering) that you said Core had multiple DLLs where Framework has mscorlib. Is it just one DLL for Core?

Reply:
It may be viewed as the root in the assemblies tree.  netstandard.dll contains only references to other assemblies.  The other assemblies contain types and references to yet other assemblies.  And so on.

Lazy resolvers later do a good job of following the references and finding required types.  But we have to start from something.  And netstandard seems to be a good starting point.